charmcraft/commands/build.py

```diff
@@ -177,41 +183,54 @@ def run(self, bases_indices: Optional[List[int]] = None) -> List[str]:
         """
         charms: List[str] = []
 
-        if is_charmcraft_running_in_managed_mode():
-            if self.config.bases is None:
-                # XXX: 2021-06-16 Patterson temporary until we have the default base
-                raise CommandError("Bases are currently required in managed-mode.")
-
-            for i, bases_config in enumerate(self.config.bases):
-                if bases_indices and i not in bases_indices:
+        if self.config.bases:
```

Comment:
Shouldn't we always have "bases"? Even if not mandatory yet, some default?

Reply:
Yes.  But that PR is upcoming.  At the moment this keeps behavior consistent.